packages/icons/general/Cram.tsx

```diff
@@ -0,0 +1,10 @@
+import { createIcon } from '../utils'
+
+export const CramIcon = createIcon(
+    'CramIcon',
+    <g fill="none">
+        <path opacity=".2" d="M12 2C6.477 2 2 6.477 2 12s4.477 10 10 10 10-4.477 10-10S17.523 2 12 2Z" fill="#F4212E" />
+        <path d="m13.637 6.644-4.633 4.633 2.453 1.635L10.367 17 15 12.367l-2.453-1.635 1.09-4.088Z" fill="#F4212E" />
```

Comment:
It's ok to replace #F4212E with redMain(#FF5F5F). I think this is a small design mistake.